openshift-hack/e2e/annotate/rules.go

```diff
@@ -26,6 +26,7 @@ var (
 			`\[Feature:RelaxedDNSSearchValidation\]`,
 			`\[Feature:PodLogsQuerySplitStreams\]`,
 			`\[Feature:PodLifecycleSleepActionAllowZero\]`,
+			`\[Feature:OrderedNamespaceDeletion\]`, // disabled Beta
```

Comment:
Can you make this commit a `carry` instead? Otherwise we'll miss this commit in the next rebase.